pom.xml

```diff
@@ -12,7 +12,7 @@
     <properties>
         <dist.dir>${project.build.directory}/dist</dist.dir>
         <java.numeric.version>1.8</java.numeric.version>
-        <kafka.version>1.1.0</kafka.version>
+        <kafka.version>2.1.0</kafka.version>
```

Comment:
This change causes a few new shader warnings.  I suggest you update the `maven-shade-plugin` to version `3.2.1 `, and also add this exclusion to the `connect-runtime` dependency:
```    
           <exclusions>
                <exclusion>
                    <groupId>org.glassfish.hk2.external</groupId>
                    <artifactId>javax.inject</artifactId>
                </exclusion>
            </exclusions>
```

This still leaves one warning about breaking module encapsulation, which seems to be unavoidable when using module dependency jars with the shader.  Long term we may need to consider a different way of packaging dependencies, but this seems to be OK for now.